sdk/compute/mgmt/src/main/java/com/azure/resourcemanager/compute/implementation/WindowsVolumeNoAADEncryptionMonitorImpl.java

```diff
@@ -155,13 +155,7 @@ public DiskVolumeEncryptionMonitor refresh() {
             .computeManager
             .inner()
             .getVirtualMachines()
-            .getByResourceGroupAsync(rgName, vmName)
-            .onErrorResume(
```

Comment:
Track1 does this

```
    private Observable<VirtualMachineInner> retrieveVirtualMachineAsync() {
        return this.computeManager
                .inner()
                .virtualMachines()
                .getByResourceGroupAsync(rgName, vmName, InstanceViewTypes.INSTANCE_VIEW)
                .flatMap(new Func1<VirtualMachineInner, Observable<VirtualMachineInner>>() {
                    @Override
                    public Observable<VirtualMachineInner> call(VirtualMachineInner virtualMachine) {
                        if (virtualMachine == null) {
                            return Observable.error(new Exception(String.format("VM with name '%s' not found (resource group '%s')",
                                    vmName, rgName)));
                        }
                        return Observable.just(virtualMachine);
                    }
                });
    }
```

Basically it convert null to Exception.

Since we already has Exception here, no need for this code.